loader/src/main/java/net/neoforged/fml/loading/targets/JUnitDevLaunchTarget.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.fml.loading.targets;
+
+import net.neoforged.api.distmarker.Dist;
+
+public class JUnitDevLaunchTarget extends CommonDevLaunchHandler {
+    @Override
+    public Dist getDist() {
+        return Dist.DEDICATED_SERVER;
```

Comment:
We need a way to load client classes too.

Reply:
The dist of the launch target doesn't change the classes being loaded, just configures the RDC. Configuration which will be overwritten by the neo bootstrap.